node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "3.3.1",
+  "version": "3.3.2",
```

Comment:
Thanks for addressing these vulns. Is there an ETA for this to drop in NPM registry?